agents-common/src/main/java/org/apache/ranger/plugin/model/RangerPolicy.java

```diff
@@ -19,664 +19,672 @@
 
 package org.apache.ranger.plugin.model;
```

Comment:
How about adding a rule in checkstyle going forward to not have a java file with X lines.

Reply:
Can you please clarify what "X lines" means?

Reply:
Sure, I meant it's subjective. I'd prefer it to be at max 1000 lines.

Reply:
ok. Makes sense. Let's consider this for next round.